mpt/impl/imap-mailbox/inmemory/pom.xml

```diff
@@ -81,7 +81,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <reuseForks>true</reuseForks>
+                    <reuseForks>false</reuseForks>
```

Comment:
This was needed to retain isolation for `MailboxPath`' static final INVALID_CHARS across test classes.

Acceptable for memory tests IMO.

Reply:
Ok. What's the overall degradation?

Reply:
On my local PC, for the MPT memory module. Test run time:
Before: 35s
After: 36s

Neglectable. Likely bootstrap new JVMs is too fast on my PC.